src/io.rs

```diff
@@ -49,7 +49,16 @@ fn eprintln(s: &str) -> IO<()> {
 
 #[derive(Userdata)]
 #[gluon(crate_name = "::vm")]
-struct GluonFile(Mutex<File>);
+struct GluonFile(Mutex<Option<File>>);
```

Comment:
Could have sworn that rust had a `File::close(&mut self)` method but it seems that is not the case :/